buildifier/README.md

```diff
@@ -26,46 +26,55 @@ in a directory recursively:
 
     $ buildifier -r path/to/dir
 
-Buildifier automatically detects the file type (either BUILD or .bzl) by its filename. If you 
-
-    $ buildifier $(find . -type f \( -iname BUILD -or -iname BUILD.bazel \))
-
-Files with unknown names (e.g. `foo.bar`) will be formatted as .bzl files because the format for
-.bzl files is more flexible and less harmful.
-
-You can use Buildifier as a filter by invoking it with no arguments. In that mode it reads from
-standard input and writes the reformatted version to standard output. In this case it won't be
-able to see its name to choose the correct formatting rules, and for compatibility reasons it
-will use the BUILD format in such situations. This may be changed in the future, and to enforce
-a special format explicitly use the `--type` flag:
+Buildifier supports the following file types: BUILD, WORKSPACE, .bzl, and
```

Comment:
Can you add backticks around all file types please?

Reply:
Done.